packages/medusa/src/services/product-variant.ts

```diff
@@ -505,11 +521,11 @@ class ProductVariantService extends BaseService {
    * @param {string} optionId - the option from product.
    * @return {Promise} empty promise
    */
-  async deleteOptionValue(variantId, optionId) {
-    return this.atomicPhase_(async (manager) => {
+  async deleteOptionValue(variantId: string, optionId: string): Promise<void> {
+    return this.atomicPhase_(async (manager: EntityManager) => {
       const productOptionValueRepo = manager.getCustomRepository(
         this.productOptionValueRepository_
-      )
+      ) as ProductOptionValueRepository
```

Comment:
Should probably be productOptionValueRepo: ProductOptionValueRepository = manager.getCusto.... to stay consistent with the way we do this in the API